### PR TITLE
add lst creation script

### PR DESCRIPTION
**Pull request by gpsanant**
script to create new lst strategies with tvl limits from config

adds date parsing for easy handling of file names

### CODE REVIEW

File: test.sh [hunk 1]
```diff
@@ -1,23 +1,22 @@
...
+declare -a addresses=(
+    "0xf1C9acDc66974dFB6dEcB12aA385b9cD01190E38"
+    "0xf951E335afb289353dc249e82926178EaC7DEd78"
+    "0x856c4Efb76C1D1AE02e20CEB03A2A6a08b0b8dC3"
+    "0xA35b1B31Ce002FBF2058D22F30f95D405200A15b"
+    "0xa2E3356610840701BDf5611a53974510Ae27E2e1"
+    "0xE95A203B1a91a908F9B9CE46459d101078c2c3cb"
+    "0xac3E018457B222d93114458476f3E3416Abbe38F"
+    "0x8c1bed5b9a0928467c9b1341da1d7bd5e10b6549"
+)
```
Comment (ChaoticWalrus) [inline test.sh:16]: this is a script that's checking token decimals. maybe clean it up?

File: script/strategy/configs/strategy_deployment_mainnet_2023_12.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            "token_address": "0x856c4Efb76C1D1AE02e20CEB03A2A6a08b0b8dC3",
+            "token_symbol": "oETH",
+            "max_per_deposit": 115792089237316195423570985008687907853269984665640564039457584007913129639935,
+            "max_deposits": 200000000000000000000000
+        },
+        {
+            "token_address": "0xA35b1B31Ce002FBF2058D22F30f95D405200A15b",
+            "token_symbol": "ETHx",
+            "max_per_deposit": 115792089237316195423570985008687907853269984665640564039457584007913129639935,
+            "max_deposits": 200000000000000000000000
```
[the remark this answers is not in the document]
Comment (ChaoticWalrus) [reply script/strategy/configs/strategy_deployment_mainnet_2023_12.json:25]: here's the lines
https://github.com/Layr-Labs/eigenlayer-contracts/blob/b599cfe200b656efc26335f0340e4012b34d2399/src/contracts/strategies/StrategyBaseTVLLimits.sol#L59-L64
maybe you are both setting and checking these in a flipped way?

File: test.sh (hunk 1 above)
Comment (gpsanant) [reply test.sh]: deleted

File: script/strategy/configs/strategy_deployment_mainnet_2023_12.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (gpsanant) [reply script/strategy/configs/strategy_deployment_mainnet_2023_12.json]: fixed!

**Review by ChaoticWalrus — APPROVED** [review]
LGTM now; thanks for this

**Pull request merged**